### PR TITLE
feat: [NODE-1499] Create and use a faster alternative to `prestorecon`

### PR DESCRIPTION
**Pull request by Bownairo**

### CODE REVIEW

**Comment by Bownairo** [comment]
**Before**:
`Reboot took: 140s`

```
# systemd-analyze blame
1min 55.745s setup-permissions.service
      4.715s systemd-networkd-wait-online.service
       604ms dev-mapper-vroot.device
       555ms reload_nftables.service
       441ms systemd-cryptsetup@vda10\x2dcrypt.service
       367ms var-lib-ic-data.mount
       346ms systemd-cryptsetup@var_crypt.service
       281ms setup-lvs.service
       198ms systemd-udev-trigger.service
       169ms setup-ssh-account-keys.service
       ...
```

**After**:
`Reboot took: 84s`

```
# systemd-analyze blame
58.825s setup-permissions.service
 4.432s systemd-networkd-wait-online.service
  597ms dev-mapper-vroot.device
  546ms reload_nftables.service
  359ms var-lib-ic-data.mount
  268ms setup-lvs.service
  252ms systemd-udev-trigger.service
  231ms systemd-cryptsetup@vda10\x2dcrypt.service
  140ms sysfsutils.service
  136ms lvm2-monitor.service
  ...
```

---

I should write a test to make sure that both are touching all of the same files.

File: ic-os/components/ic/setup-permissions/erestorecon.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+# erestorecon (easy prestorecon) uses UNIX tools to parallelize restorecon,
+# instead of the cpp based prestorecon.
+
+find $@ -print0 | xargs -0 -P 0 restorecon
```
Comment (frankdavid) [inline ic-os/components/ic/setup-permissions/erestorecon.sh:8]: Elegant!

**Review by frankdavid — APPROVED** [review]
So elegant, I love it!

**Comment by Bownairo** [comment]
Checking directly

`prestorecon`:
```
Processed: 1001041 Relabeled: 0 Elapsed: 79040ms
```

`erestorecon`:
(`find /var/lib/ic/data /var/lib/ic/crypto -print0 | xargs -0 -P 0 -t restorecon 2>/tmp/out`)
```
# wc -l /tmp/out
320 /tmp/out
```
So there are 320 batches...

```
# wc -w /tmp/out
1001361 /tmp/out
```

...and 1001361 - 320 = 1001041 total inputs.

**Pull request merged**